src/accordion/accordion.ts

```diff
@@ -45,27 +17,48 @@ export class NgbAccordion {
     </div>
   `
 })
-export class NgbAccordionGroup implements OnDestroy {
-  private isDisabled: boolean;
-  private _isOpen: boolean = false;
+export class NgbAccordionGroup {
+  private _isOpen = false;
+  private accordion: NgbAccordion;
+  @Input() isDisabled: boolean;
+  @Input() heading: string;
```

Comment:
I think that we should use the `title` here as this is what users can change here. `heading` should be reserved for changing the entire heading (which we don't have yet)